performance/images.md

```diff
@@ -1,22 +1,43 @@
 
-In a frontend workflow we typically deal with two kinds of image:
+## Introduction
+
+In a frontend workflow we typically deal with two different categories of images:
+
+* **User interface images**. These include icons, logos, buttons, and any other image that is part of the user. Usually managed by the frontend, these are stored with other client-side resources such as templates, CSS and JS. Like those resources, they may be shared between many different pages.
+
+* **Content images**. These are unique to a page, or a handful of pages. They will usually be provided from a database or a content management system.
 
-* **User interface images**, including icons, logos, and buttons. Usually managed by the frontend, these are stored with other client-side resources such as templates, CSS and JS. Like those resources, they may be shared between many different pages.
+Most best practices described in this document apply to both categories unless otherwise indicated.
 
-* **Content images**. These are unique to a page (or a handful of pages). They will usually be provided from a database or a content management system.
+## Picking the right image format
 
-Most best practices are common to both categories.
+* Use [SVG](https://en.wikipedia.org/wiki/Scalable_Vector_Graphics) for icons, logos, and other images with sharply defined edges.
+* Use [JPEG](https://en.wikipedia.org/wiki/JPEG) for still images, photographs, and in any other cases where using a PNG would result in an image too large to be delivered over the network effectively.
+* Use [PNG](https://en.wikipedia.org/wiki/Portable_Network_Graphics) for illustrations, graphs, flat-colour graphics, images containing transparency (which JPEG doesn't support), and images containing text.
+* Use [MP4](https://en.wikipedia.org/wiki/MPEG-4_Part_14) for any kind of animated images.
+* Do not use [GIFs](https://en.wikipedia.org/wiki/GIF). The file size of animated GIF images is usually an order of magnitude bigger than MP4s of similar quality. They can be a barrier that prevents users from accessing our content, especially in markets where slower connections and low performing devices are more common. For static images with transparency, an 8-bit PNG can be used to deliver an indexed colour image, with binary transparency, [at a smaller size than a GIF](https://helpx.adobe.com/photoshop-elements/using/optimizing-images-gif-or-png.html).
 
-## Formats
+A number of more modern image formats can also be used to deliver images:
 
-### Which format to use
+* [WEBP](https://en.wikipedia.org/wiki/WebP) is a replacement for JPEG, PNG, and GIFs, but improvements to encoders like mozjpeg means that WEBP file sizes are only marginally smaller than JPEG. It's supported by almost all major browsers ([Safari supports it partially](https://caniuse.com/webp)) and can be used by providing another better supported format as a fallback.
```

Comment:
```suggestion
* [WEBP](https://en.wikipedia.org/wiki/WebP) is a replacement for JPEG, PNG, and GIFs, but improvements to encoders like mozjpeg means that WEBP file sizes are only marginally smaller than JPEG. It's supported by almost all major browsers ([Safari supports it partially](https://caniuse.com/webp)) and can be used only as an enhancement to another better supported format.
```
Using the language of progressive enhancement rather than graceful degradation.